packages/vuetify/src/components/VSelect/VSelect.ts

```diff
@@ -183,20 +182,20 @@ export default VTextField.extend({
         }
       }
     },
-    staticList () {
+    staticList (): VNode {
       if (this.$slots['no-data'] || this.$slots['prepend-item'] || this.$slots['append-item']) {
         consoleError('assert: staticList should not be called if slots are used')
       }
 
       return this.$createElement(VSelectList, this.listData)
     },
-    virtualizedItems () {
+    virtualizedItems (): object[] {
       return this.$_menuProps.auto
         ? this.computedItems
         : this.computedItems.slice(0, this.lastItem)
     },
-    menuCanShow () { return true },
-    $_menuProps () {
+    menuCanShow (): boolean { return true },
+    $_menuProps (): any {
```

Comment:
`: object`?

Reply:
@jacekkarczmarczyk It will cause many errors like prop isn't defined in type ....